Morgan-Stanley/testplan
# Redesign how interactive runner runs tests

Implement new design for the interaction between the interactive runner
and individual MultiTests to run tests:

* Add new method to MultiTest to run testscases iteratively and yield
  individual testcase reports as they are run. The interactive runner
  handles merging testcase reports into its main report tree. This way,
  setup and teardown methods are only run once per suite and return
  appropriate report entries that can be rendered in the UI.
* Remove separate MultiTestIRunner class that is no longer required.
* Remove run buttons from special "suite related" report entries such as
  testsuite setup/teardown. These methods are run automatically by the
  backend as required and it does not really make sense for them to be
  manually run on their own.
* Small aesthetic tweak to UI to use a spinning icon instead of a bar
  loader when running tests.
